src/ViewModels/DiffContext.cs

```diff
@@ -66,6 +68,17 @@ public Vector SyncScrollOffset
             set => SetProperty(ref _syncScrollOffset, value);
         }
 
+        public Models.FileModeDiff FileModeDiff
+        {
+            get => _fileModeDiff;
+            set => SetProperty(ref _fileModeDiff, value);
+        }
+
+        public TextTrimming PathTrimming
```

Comment:
PathConverters.TruncateIfTooLong is this way on purpose that is to hide in the middle of the path, but retained 20 characters at the beginning of the path and the entire file name in the end.

Reply:
![image](https://github.com/sourcegit-scm/sourcegit/assets/28685179/9addfdea-f62d-40c9-a5ba-927e15818d05)

I use textTrimming because I want to avoid this

Reply:
Yes. Perhaps I should re-design the toolbar for DiffView.